frontend/packages/ceph-storage-plugin/src/utils/bucket-class.ts

```diff
@@ -14,3 +14,8 @@ export const validateDuration = (ms: number): boolean => ms >= 900000 && ms <= 8
 
 export const convertToMS = ({ unit, value }) =>
   unit === TimeUnits.HOUR ? parseInt(value, 10) * 3600000 : parseInt(value, 10) * 60000;
+
+export const convertMsToUnit = (unit: string, value: number): number =>
+  unit === TimeUnits.HOUR ? value / 3600000 : value / 60000;
+
+export const setTimeUnit = (unit: string): string => (unit === TimeUnits.HOUR ? 'hr' : 'min');
```

Comment:
nit-pick:
```suggestion
export const getTimeUnit = (unit: string): string => (unit === TimeUnits.HOUR ? 'hr' : 'min');
```
Also I think this can be a constant object/enum instead of a function.

Reply:
Yes I was thinking a same. This is more of a map.